torch/testing/_internal/common_methods_invocations.py

```diff
@@ -2463,6 +2486,22 @@ def gradcheck_wrapper_triangular_input(op, input, *args, upper=False, **kwargs):
                # TODO: remove redundant method_tests() entries
                SkipInfo('TestOpInfo', 'test_duplicate_method_tests')),
            sample_inputs_func=sample_inputs_addmm),
+    OpInfo('addmv',
+           dtypes=floating_types(),
+           dtypesIfCPU=all_types_and_complex_and(torch.bfloat16),
+           dtypesIfCUDA=floating_types_and(torch.float16, torch.complex64, torch.complex128,
+                                           *[torch.bfloat16] if CUDA11OrLater else []),
+           dtypesIfROCM=floating_types_and(torch.half),
+           supports_inplace_autograd=False,
+           skips=(
+               # https://github.com/pytorch/pytorch/issues/55539 to track all issues for addmv
+               # failed with a large difference in tensor outputs for below tests
+               SkipInfo('TestCommon', 'test_variant_consistency_eager',
```

Comment:
I'm curious if working around the addmv_ bug will make this skip no longer necessary